test/unit/app/controllers/transactions/tx-controller-test.js

```diff
@@ -183,74 +175,55 @@ describe('Transaction Controller', function () {
       getPermittedAccounts.restore()
     })
 
-    it('should add an unapproved transaction and return a valid txMeta', function (done) {
-      txController.addUnapprovedTransaction({ from: selectedAddress })
-        .then((txMeta) => {
-          assert(('id' in txMeta), 'should have a id')
-          assert(('time' in txMeta), 'should have a time stamp')
-          assert(('metamaskNetworkId' in txMeta), 'should have a metamaskNetworkId')
-          assert(('txParams' in txMeta), 'should have a txParams')
-          assert(('history' in txMeta), 'should have a history')
-
-          const memTxMeta = txController.txStateManager.getTx(txMeta.id)
-          assert.deepEqual(txMeta, memTxMeta, `txMeta should be stored in txController after adding it\n  expected: ${txMeta} \n  got: ${memTxMeta}`)
-          done()
-        }).catch(done)
+    it('should add an unapproved transaction and return a valid txMeta', async function () {
+      const txMeta = await txController.addUnapprovedTransaction({ from: selectedAddress })
+      assert.ok(('id' in txMeta), 'should have a id')
+      assert.ok(('time' in txMeta), 'should have a time stamp')
+      assert.ok(('metamaskNetworkId' in txMeta), 'should have a metamaskNetworkId')
+      assert.ok(('txParams' in txMeta), 'should have a txParams')
+      assert.ok(('history' in txMeta), 'should have a history')
```

Comment:
nit: style: compare with `'should return a state object with the right keys and data types'` above, those "in" statements are not parenthesized. I'm agnostic as to which we do, but we should probably stick with one in the name of consistency.

Reply:
Ehh these are the types of things we should have lint rules for. I'll update these lines since I've already changed them but we should update our [`no-extra-parens`](https://eslint.org/docs/rules/no-extra-parens) rule